01-js-intro-and-jquery/08/js/scripts.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready(function() {
+	
+	var alias = '';
+	
+	var focusAlias = function() {	
+		$('.alias').focus(); 
+	};
+	
+	var getResponse = function() {
+		alias = $('.alias').val();
+
+		$.ajax({
+   url: "http://bootcamp.aws.af.cm/welcome/" + alias,
+   type: 'GET',
+   dataType: "json",
+   success: function( data ) {
+   	if(data.error) {
+  			$('#response').html(data.error);
```

Comment:
This is not incorrect but depends on your response sending an ``error`` property in your description. I think this should be handled in your ``error`` callback.